src/dashboard/infrastructure/integrations/site-kit.php

```diff
@@ -61,6 +64,51 @@ public function is_enabled(): bool {
 		return \is_plugin_active( self::SITE_KIT_FILE );
 	}
 
+	/**
+	 * If the Google site kit setup has been completed.
+	 *
+	 * @return bool If the Google site kit setup has been completed.
+	 */
+	private function is_setup_completed() {
+		return \get_option( 'googlesitekit_has_connected_admins', false ) === '1';
+	}
+
+	/**
+	 * If consent has been granted.
+	 *
+	 * @return bool If consent has been granted.
+	 */
+	private function is_connected() {
+		return $this->site_kit_consent_repository->is_consent_granted();
+	}
+
+	/**
+	 * If Google analytics is connected.
+	 *
+	 * @return bool If Google analytics is connected.
+	 */
+	public function is_ga_connected() {
+		return $this->site_kit_analytics_4_adapter->is_connected();
+	}
+
+	/**
+	 * If the Site Kit plugin is installed.
+	 *
+	 * @return bool If the Site Kit plugin is installed.
+	 */
+	private function is_site_kit_installed() {
```

Comment:
Lets mention somewhere that this is because we can't use \is_plugin_active`()` in REST requests.

If not in the name of the function, at least in the function description above